tests/visual/project.spec.ts

```diff
@@ -5,9 +5,16 @@ import {
   aliceRemote,
 } from "@tests/support/fixtures.js";
 
-test("source tree page", async ({ page }) => {
-  await page.goto(sourceBrowsingUrl, { waitUntil: "networkidle" });
-  await expect(page).toHaveScreenshot();
+test.describe(async () => {
+  test("source tree page", async ({ page }) => {
+    await page.goto(sourceBrowsingUrl, { waitUntil: "networkidle" });
+    await expect(page).toHaveScreenshot();
+  });
+  test.use({ viewport: { width: 414, height: 896 } });
```

Comment:
I think it's best to have a separate folder in `tests/visual/mobile` and copy the project spec there and run all the tests with a custom viewport.

We can do this for all the other tests as well, but that can be a follow up (see https://github.com/radicle-dev/radicle-interface/issues/850).

It seems like `test.use` applies to the whole test otherwise:

<img width="766" alt="Screenshot 2023-06-19 at 14 46 22" src="https://github.com/radicle-dev/radicle-interface/assets/158411/95602f43-a10a-4ccd-b4bb-9a52cd84b90f">

Reply:
Which phone is the custom size `{ width: 414, height: 896 }`? Maybe we can extract that into a variable and use it for the other mobile tests.

Reply:
thats an iPhoneXR according to chrome, can extract the size into a variable

Reply:
Ok moved the mobile test into a separate folder.
Would prefer to move the other tests in a separate PR.

Reply:
I think the other alternative is to wrap all tests in a file in a top level `describe` scope and define the screen size there.  No need to change anything.

```
describe "mobile" {
  use mobile-screen size

  test-1
  test-2
  ...
}
describe "desktop" {
  use mobile-screen size

  test-1
  test-2
  ...
}
```

Reply:
hmm ok yeah makes sense 👍